setup.py

```diff
@@ -24,9 +26,12 @@
     'requests',
     'six',
     'structlog',
-    'ujson',
 ]
 
+# ujson is not pypy compliant, as it uses the CPython C API
+if not installed_with_pypy:
```

Comment:
If we do that for ``ujson``, we can probably do it as well for ``psycopgcffi`` in the ``cliquet[postgresql]`` rule